test/dataloader/ecto/limit_query_test.exs

```diff
@@ -42,6 +43,7 @@ defmodule Dataloader.LimitQueryTest do
     schema
     |> order_by(^order_by)
     |> limit(^limit)
+    |> distinct(true)
```

Comment:
Can we make a dedicated clause for this? With this change every clause is distinct, which would make test cases showing that we _add_ a distinct no longer valid.